src/DecoratorManager.js

```diff
@@ -51,7 +51,7 @@ DecoratorManager.prototype.new = function(options, callback) {
 
         // Get the src, test paths
         var setupConfig = R.mapObjIndexed(function(empty, type) {
-            return path.join(type, 'decorators', name+'Decorator');
+            return `${type}/decorators/${name}Decorator`;
```

Comment:
I am using ES6 and using babel to cross compile down to es5. These are ES6 string templates - basically they are strings with interpolation. ie, `'hello ' + name + ', my name is ' + me` is the same as `'hello ${name}, my name is ${me}'`